tests/browser/stn/stn-resource-timing.browser.js

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright 2020 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+let test = require('../../../tools/jil/browser-test.js')
+let drain = require('../../../agent/drain')
+
+require('../../../feature/stn/instrument')
+require('../../../feature/xhr/instrument')
+require('../../../feature/xhr/aggregate')
+require('../../../feature/err/instrument')
+require('../../../feature/err/aggregate')
+
+drain('api')
+
+if ((window.performance &&
+  window.performance.timing &&
+  window.performance.getEntriesByType)) {
+  runTests()
+} else {
+  test('unsupported browser', function (t) {
+    t.skip('skipping tests because browser does not have perf timing api')
+    t.end()
+  })
+}
+
+// create session trace nodes for load events
+document.addEventListener('DOMContentLoaded', () => null)
+window.addEventListener('load', () => null)
+
+function runTests () {
+  let ee = require('ee')
+
+  test('wait for trace node generation', function (t) {
```

Comment:
I don't think we need all this setup since the test below is not looking for it. It should generate some resource data that is used below.